frontend-v2/src/lib/auth.ts

```diff
@@ -0,0 +1,5 @@
+import { headers } from 'next/headers'
+
+export async function getAuthCookies() {
```

Comment:
nit: name is misleading imo since this is just returning the full cookie header & not really anything specific to auth